@xen-orchestra/web/src/components/backups/configuration/BackupJobSettings.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <UiCard>
+    <UiTitle> {{ t('settings') }} </UiTitle>
+    <VtsColumns>
+      <VtsColumn>
+        <VtsQuickInfoRow :label="t('proxy')" :value="proxy?.name" />
+        <VtsQuickInfoRow :label="t('snapshot-mode')">
+          <template v-if="backupJob.mode" #value>
+            <UiTag variant="secondary" accent="info">{{ backupJob.mode }}</UiTag>
+          </template>
+        </VtsQuickInfoRow>
+        <VtsQuickInfoRow
+          :label="t('vm-backup-failure-number-of-retries')"
+          :value="settings.nRetriesVmBackupFailures ? String(settings.nRetriesVmBackupFailures) : undefined"
```

Comment:
You can add an explicit check on `undefined`, so when this setting is explicitly set to `0`, it would be correctly displayed, like in the side panel on the backup jobs page:
```suggestion
          :value="settings.nRetriesVmBackupFailures !== undefined ? String(settings.nRetriesVmBackupFailures) : undefined"
```